tests/nist/Makefile.am

```diff
@@ -1,7 +1,7 @@
 EXTRA_DIST = $(top_srcdir)/tests/test_common.sh
 
-DISTCLEANFILES = *.log *.results oscap_debug.log.* *.result.xml *.results_arf.xml */*.results_oval.xml */*.stdout
-CLEANFILES = *.log *.results oscap_debug.log.* *.result.xml *.results_arf.xml */*.results_oval.xml */*.stdout
+DISTCLEANFILES = *.log *.results oscap_debug.log.* *.result.xml */*.results_arf.xml */*.results_oval.xml */*.stdout
+CLEANFILES = *.log *.results oscap_debug.log.* *.result.xml */*.results_arf.xml */*.results_oval.xml */*.stdout
```

Comment:
Nice side effect is that it cleans output put in specified `--outputdir` if it is inside `tests/nist` directory.
I guess some people might not expect it.